packages/mattermost-redux/src/client/queries.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const myDataQueryString = `
```

Comment:
move this to `mattermost-redux/actions/users_query`
And add the types in the same file